LLama/Native/NativeApi.Load.cs

```diff
@@ -57,6 +57,11 @@ private static void SetDllImportResolver()
             // NativeLibrary is not available on older runtimes. We'll have to depend on
             // the normal runtime dll resolution there.
 #if NET5_0_OR_GREATER
+
+            // We don't need special dll resolution on Android
+            if (OperatingSystem.IsAndroid())
```

Comment:
The master branch has refactored the native library loading system so I'm afraid this code would not work. Don't worry, I can do it after having this PR merged.